server/command_create.go

```diff
@@ -140,9 +140,11 @@ func (p *Plugin) runCreateCommand(args []string, extra *model.CommandArgs) (*mod
 
 	database := ""
 	if install.Database == databaseOptionRDS {
-		database = cloud.InstallationDatabaseAwsRDS
+		database = cloud.InstallationDatabaseSingleTenantRDS
```

Comment:
nit: I think you could use switch here but no worries if this is just a quick change.